dfx.json

```diff
@@ -63,6 +63,12 @@
       "type": "assets",
       "source": ["apps/wallet/dist/"],
       "build": "pnpm --filter 'wallet-dapp' build"
+    },
+    "wasm_chunk_store": {
+      "type": "custom",
```

Comment:
can this not be something simple like

```
"wasm_chunk_store": {
      "type": "assets",
      "source": []
    },
```

not sure if this is correct dfx.json, but this would let you not need to download the wasm/candid.

Reply:
Not sure why, but the frontend fails to load if I do this change.

Reply:
It looks like a dfx bug to me - this dfx.json file in a sample project created by dfx new
```
{
  "canisters": {
    "test_backend": {
      "main": "src/test_backend/main.mo",
      "type": "motoko"
    },
    "test_frontend": {
      "dependencies": [
        "test_backend"
      ],
      "source": [
        "src/test_frontend/dist"
      ],
      "type": "assets",
      "workspace": "test_frontend"
    },
    "test": {
      "type": "assets",
      "source": []
    }
  },
  "defaults": {
    "build": {
      "args": "",
      "packtool": ""
    }
  },
  "output_env_file": ".env",
  "version": 1
}
```
results in `dfx deploy --no-wallet` being flaky which is a sign of a bug in my opinion.

Reply:
not sure if that bug is related to our issue though

Reply:
Adding `"build": ""` fixed it for me. Could you please confirm the local deployment works for you on my recent commit, @olaszakos ?

Reply:
I can confirm that local deployment works now.